.github/workflows/test.yml

```diff
@@ -379,31 +379,16 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - uses: ./.github/actions/setup-turborepo-environment
-        with:
-          target: ${{ matrix.os.name }}
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-
-      - name: Setup Pnpm
-        uses: pnpm/action-setup@v2.2.4
+      - name: "Setup Node"
+        uses: ./.github/actions/setup-node
         with:
-          version: 7.2.1
+          enable-corepack: "false"
+          package-install: "false"
+          enable-pnpm-cache: "true"
 
-      - name: Make sure pnpm always has a cache
+      - name: Install Turbo globally
         shell: bash
-        run: |
-          mkdir -p `pnpm store path`
-
-      - name: Disable corepack
-        shell: bash
-        run: corepack disable
-
-      - name: Setup Node.js
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16
-          cache: ${{ matrix.manager }}
-          cache-dependency-path: package.json
+        run: npm i -g turbo@canary
```

Comment:
I agree that our existing setup is weird and this is basically no different from our existing setup unless we make changes in the global=>local delegation.

But why would we choose to do this instead of `--skip-infer` and use the `HEAD` version directly for executing the examples?

Reply:
examples tests are testing the _examples_, not the cli. They don't even run when the cli source changes. They're meant to test that an out-of-box example works. This is noted in the README too: https://github.com/vercel/turbo/blob/main/turborepo-tests/examples/README.md

I agree it would be nice to utilize examples to test `HEAD` of cli as well, but this PR is not the thing that will get us there. (nudge: read the rfc I wrote months ago and help move it forward 😃)